dspace-server-webapp/src/main/java/org/dspace/app/rest/link/process/ProcessHalLinkFactory.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.link.process;
+
+import org.dspace.app.rest.ProcessRestController;
+import org.dspace.app.rest.link.HalLinkFactory;
+
+public abstract class ProcessHalLinkFactory<T> extends HalLinkFactory<T, ProcessRestController> {
```

Comment:
Needs JavaDocs to describe this abstract class.  This is especially important because it's an empty class, so we need to describe why it's needed.